cnxepub/html_parsers.py

```diff
@@ -16,6 +16,15 @@
     }
 
 
+def _squash_to_text(elm):
+    value = [elm.text or '']
+    for child in elm.getchildren():
+        value.append(etree.tostring(child).decode('utf-8'))
+        value.append(child.tail or '')
+    value = ''.join(value).encode('utf-8')
+    return value
+
+
```

Comment:
this is basically the same as etree.tostring(elm, method=text, with_tail=False) no?